index.html

```diff
@@ -60,9 +60,296 @@ <h2>I`m a sofware developer</h2>
+              <div class="backabouttwo"><img src="/img/backgound-2-about-me.svg" alt=""></div>
+              
+              <div class="bg">
+                <button class="goproject" type="button">Get My Resume</button>
               </div>
+            
+            </section>
+
+            <ul>
+              <li class="tech-card">
+                <div class="img"></div>
+                <div class="about-me-icons">
+                  <div><img src="/Offline-Icons/about-me-icons.-1.svg" alt=""></div>
+                </div>
+      
+                <h3>Languages</h3>
+      
+                <ul class="mini-card-list-l">
+                  <li class="list-name-i">
+                    Javascript
+                  </li>
+                  <li class="list-name-i">
+                    Ruby
+      
+                  </li>
+                  <li class="list-name-i">
+                    Html
+                  </li>
+                  <li class="list-name-i">
+                    Css 
+                  </li>
+                </ul>
+      
+              </li>
+            
+              <li class="tech-card">
+                <div class="img"></div>
+                <div class="about-me-icons">
+                  <img src="/Offline-Icons/about-me-icons-2.svg" alt="">
+                </div>
+      
+                <h3>Frameworks</h3>
+      
+                <ul class="mini-card-list-l">
+                  <li class="list-name-i">
+                    Bootstrap
+                  </li>
+                  <li class="list-name-i">
+                    Ruby on rails
+                  </li>
+                  <li class="list-name-i">
+                    RSspec
+                  </li>
+                  <li class="list-name-i">
+                    Capybara
+                  </li>
+                  <li class="list-name-i">
+                    Selenium
+                  </li>
+                </ul>
+      
+              </li>
+      
+              <li class="tech-card">
+                <div class="img"></div>
+                <div class="about-me-icons">
+                  <img src="/Offline-Icons/about-me-icons-3.svg" alt="">
+                </div>
+      
+                <h3>Skills</h3>
+      
+                <ul class="mini-card-list-l">
+                  <li class="list-name-i">
+                    Codekit
+                  </li>
+                  <li class="list-name-i">
+                    Github
+                  </li>
+                  <li class="list-name-i">
+                    Codepen
+                  </li>
+                  <li class="list-name-i">
+                    Gitlab
+                  </li>
+                  <li class="list-name-i">
+                    Terminal
+                  </li>
+                </ul>
+      
+              </li>
+            </ul>
```

Comment:
- [ ] Kindly make sure all the **about me** is wrapped in a `<section>` tag as their container. As you can see the technology categories ("Languages", "Frameworks", and "Skills") cards are outside the about me section. This will make your code follow good semantics of HTML, be more professional, and as it is required for the project.I suggest you insert all cards inside the `<section>` tag as their main container.